Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts

```diff
@@ -50,5 +51,5 @@ app.use((err, req, res, next) => { // eslint-disable-line @typescript-eslint/no-
 app.set('port', process.env.PORT || 3000);
 
 const server = app.listen(app.get('port'), function () {
-    debug('Express server listening on port ' + server.address().port);
+    debug('Express server listening on port ' + (server.address() as AddressInfo).port);
```

Comment:
If this were DT, I'd point out that `import * as debug` does not result in a callable `debug`.

Reply:
good catch!